test/format/format.sh

```diff
@@ -199,15 +204,53 @@ success=0
 running=0
 status="format.sh-status"
 
+skip_known_errors()
+{
+	# Return if "skip_errors" is not set or -E option is not passed
+	[[ $skip_errors -ne 1 ]] && return 1
+
+	log=$1
+
+	# Define each array with multi-signature matching for a single known error
+	# and append it to the skip_error_list
+	err_1=("heap-buffer-overflow" "__split_parent")
+	err_2=("heap-use-after-free" "__wt_btcur_next_random")
+	err_3=("__cursor_page_pinned" "current_state == WT_REF_LIMBO || current_state == WT_REF_MEM")
+	err_4=("cache clean check: no" "cache dirty check: no" "format run more than 15 minutes past the maximum time")
+
+	skip_error_list=( err_1[@] err_2[@] err_3[@] err_4[@] )
+
+	# Loop through the skip list and search in the log file.
+	count=${#skip_error_list[@]}
+	for ((i=0; i<$count; i++))
+	do
+		str_1=${!skip_error_list[i]:0:1}
+		str_2=${!skip_error_list[i]:1:1}
+		str_3=${!skip_error_list[i]:2:1}
```

Comment:
I guess `str_x` here means issue signature (mentioned in the comments). Could the variable names and comments be made consistent?